### PR TITLE
Forgot to add ink

### PR DESCRIPTION
**Pull request by GouravSardana**
You can review the proposal link and will gives yours suggestions.

### CODE REVIEW

**Comment by GouravSardana** [comment]
@GeorgLink You can review the link now.

File: GSoC-interest.md [hunk 1]
```diff
@@ -25,3 +25,6 @@ GSoC Mentors
 | Jaishree Dhage | jaishree.babu96 at gmail.com | Reporting of CHAOSS Metrics | [jaishreedhage/Chaoss](https://github.com/jaishreedhage/Chaoss) | To-Do |
 | Apoorv Khare | apoorvkhare007 at gmail.com | Reporting of CHAOSS Metrics | [apoorvkhare07/Chaoss-Microtasks](https://github.com/apoorvkhare07/Chaoss-Microtasks)| To-Do |
 | Anvesh Chaturvedi | anveshc10047 at gmail.com | Support of Standard Formats for Description of Projects | [anveshc05/Chaoss-Microtasks](https://github.com/anveshc05/Chaoss-Microtasks) | To-Do |
+| Gourav Sardana | gsardana001 at gmail.com | Prototype New CHAOSS Metrics | [GouravSardana/Chaoss-Microtasks](https://github.com/GouravSardana/chaoss-microtask) | https://docs.google.com/document/d/143LFq2PYXDfBQXoeBpdkP0LYdFuMCeSYA1f-0-ul9Uw/edit?usp=sharing |
+| Aman Deep Singh | ads6may at gmail.com | Reporting of CHAOSS Metrics | [ad71/chaoss-microtasks](https://github.com/ad71/chaoss-microtasks) | To-Do |
```
Comment (jgbarah) [inline GSoC-interest.md]: @GouravSardana , it seems your entry is the first one. Please, remove the change on Aman Deep Singh and Akshaht Chhabra. Please, also stash all commits: one commit should be enough for your change. Thanks!

Comment (jgbarah) [reply GSoC-interest.md]: And very likely you better rebase, because there are some more other changes to that file now.

**Comment by GouravSardana** [comment]
I think its alright now. And you can review my link & you can add comments to my doc file for changes which you want.

File: GSoC-interest.md [hunk 2]
```diff
@@ -25,10 +25,4 @@ GSoC Mentors
 | Jaishree Dhage | jaishree.babu96 at gmail.com | Reporting of CHAOSS Metrics | [jaishreedhage/Chaoss](https://github.com/jaishreedhage/Chaoss) | To-Do |
 | Apoorv Khare | apoorvkhare007 at gmail.com | Reporting of CHAOSS Metrics | [apoorvkhare07/Chaoss-Microtasks](https://github.com/apoorvkhare07/Chaoss-Microtasks)| To-Do |
 | Anvesh Chaturvedi | anveshc10047 at gmail.com | Support of Standard Formats for Description of Projects | [anveshc05/Chaoss-Microtasks](https://github.com/anveshc05/Chaoss-Microtasks) | To-Do |
-| Gourav Sardana | gsardana001 at gmail.com | Prototype New CHAOSS Metrics | [GouravSardana/Chaoss-Microtasks](https://github.com/GouravSardana/chaoss-microtask) | To-Do |
-| Aman Deep Singh | ads6may at gmail.com | Reporting of CHAOSS Metrics | [ad71/chaoss-microtasks](https://github.com/ad71/chaoss-microtasks) | To-Do |
```
Comment (GeorgLink) [inline GSoC-interest.md]: Please rebase to not remove newer entries in this file.

https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request

**Comment by GeorgLink** [comment]
This pull request should only change at most one line.

**Comment by GouravSardana** [comment]
@jgbarah kindly provide your email for contact regarding implementation of metrics

File: GSoC-interest.md [hunk 3]
```diff
@@ -24,6 +24,8 @@ GSoC Mentors
 | Sai Varun Kolluru | saivarunkolluru at gmail.com | Reporting of CHAOSS Metrics | [f2014169/Chaoss-Micro-Tasks](https://github.com/f2014169/Chaoss-Micro-Tasks) | To-Do |
 | Jaishree Dhage | jaishree.babu96 at gmail.com | Reporting of CHAOSS Metrics | [jaishreedhage/Chaoss](https://github.com/jaishreedhage/Chaoss) | To-Do |
 | Apoorv Khare | apoorvkhare007 at gmail.com | Reporting of CHAOSS Metrics | [apoorvkhare07/Chaoss-Microtasks](https://github.com/apoorvkhare07/Chaoss-Microtasks)| To-Do |
+| Anvesh Chaturvedi | anveshc10047 at gmail.com | Support of Standard Formats for Description of Projects | [anveshc05/Chaoss-Microtasks](https://github.com/anveshc05/Chaoss-Microtasks) | To-Do |
```
Comment (GeorgLink) [inline GSoC-interest.md]: Why are you adding someone other than yourself?

**Pull request closed without merging**